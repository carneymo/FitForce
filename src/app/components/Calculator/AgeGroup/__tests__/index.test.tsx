import * as React from 'react';
import { render } from '@testing-library/react';

import { AgeGroup } from '..';

describe('<AgeGroup  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<AgeGroup age={''} gender={''} />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
