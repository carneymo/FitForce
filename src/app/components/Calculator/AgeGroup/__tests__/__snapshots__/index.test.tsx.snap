// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AgeGroup  /> should match snapshot 1`] = `
<div
  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-1ky0k5h-MuiGrid-root"
>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiChip-root MuiChip-filled MuiChip-sizeSmall MuiChip-colorDefault MuiChip-filledDefault css-1fj5pxn-MuiChip-root"
    >
      <span
        class="MuiChip-label MuiChip-labelSmall css-wjsjww-MuiChip-label"
      >
        Age Group: 
      </span>
    </div>
  </div>
</div>
`;
